packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -27,6 +27,23 @@ import 'text_field.dart';
 ///    integration.
 ///  * [InputDecorator], which shows the labels and other visual elements that
 ///    surround the actual text editing widget.
+///
+/// ## Sample code
+///
+/// ```dart
+/// new TextFormField(
+///   decoration: const InputDecoration(
+///     icon: const Icon(Icons.person),
+///     hintText: 'What do people call you?',
+///     labelText: 'Name *',
+///   ),
+///   onSaved: (String value) {},
+///   validator: (String value) {
+///     if (value.contains("@")) return "Do not use the @ char.";
```

Comment:
Break the `return` onto the next line, or alternatively you could use a ternary:
```
return value.contains('@') ? 'Input may not contain @.' : null;
```